.eslintrc

```diff
@@ -1,7 +1,4 @@
-extends: universal-search
-
-parser: babel-eslint
+extends: universal-search/content
 
 rules:
-  no-console: 0
-  no-var: 2
+  func-names: 0
```

Comment:
I disabled this rule, but there really only 4 warnings generated by it currently:
```
/Users/pdehaan/dev/github/universal-search-content/src/adapters/search_suggestions_adapter.js
  26:70  warning  Missing function expression name  func-names
  29:68  warning  Missing function expression name  func-names

/Users/pdehaan/dev/github/universal-search-content/src/views/base_view.js
  44:49  warning  Missing function expression name  func-names

/Users/pdehaan/dev/github/universal-search-content/src/views/index_view.js
  68:38  warning  Missing function expression name  func-names
```